linter_exclusions.yml

```diff
@@ -1188,6 +1188,16 @@ config unset:
     key:
       rule_exclusions:
       - no_positional_parameters
+config parampersist show:
```

Comment:
why don't we use param-persist? all names of your functions is `param_persist`

Reply:
The name is provided PM after a long time survey.

Reply:
It's a bit hard to read.

Reply:
hard to understand the command name. prefer `param-persist`

Reply:
Yeah, I will confirm with PM whether we can update it to `param-persist`.